Doc/library/functions.rst

```diff
@@ -1004,7 +1004,7 @@ are always available.  They are listed here in alphabetical order.
    ``'b'``   binary mode
    ``'t'``   text mode (default)
    ``'+'``   open a disk file for updating (reading and writing)
-   ``'U'``   :term:`universal newlines` mode (deprecated)
+   ``'U'``   equivalent to ``newline=None`` (deprecated, use *newline* option)
```

Comment:
For me is a good improved. How say @csabella, it would be great have a link to *newline* definition

Reply:
Perhaps it would be clearer to put `:deprecated:` before the rest of the text
```
``'U'``   :deprecated: Use ``newline`` option instead. Equivalent to ``newline=None``
```

Having a link as suggested by @csabella would be great.